app/Http/Controllers/Api/ReactionController.php

```diff
@@ -61,7 +63,15 @@ public function store(ReactionRequest $request)
             }
         }
 
-        $totalReactions = $this->getTotalPostReactions($postableId, $postableType);
+        $reactionableType = explode('\\', $reactionableType)[2];
+
+        // @TODO: as we add more post types, we should break the below into a helper function and add different cases.
+        if ($reactionableType === 'Photo') {
+            $photo = Photo::where('id', $reactionableId)->first();
+            $totalReactions = count($photo->reactions);
+        } else {
+            throw new HttpException(405, 'Not a valid post type');
```

Comment:
It'd make sense to validate this as part of the rules in your `ReactionRequest` form request class:

```diff
+            'reactionable_id' => 'required|int',
-            'reactionable_type' => 'required',
+            'reactionable_type' => 'required|in:photo',
```